pages/linux/whatis.md

```diff
@@ -10,6 +10,10 @@
 
 `whatis --long {{command}}`
 
+- Display descriptions for all commands matching a glob:
+
+`whatis --wildcard {{command_glob}}`
```

Comment:
yes, I agree a concrete example is better in this case. I wonder if we should use more common command names for the search patterns instead, though.